app/assets/javascripts/hxlstats.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```

Comment:
Pretty sure these were scaffolded, so we can get rid of any unused files. Will mark any below that we can delete.